src/Network/Socket.cpp

```diff
@@ -29,7 +29,11 @@ namespace Stardust::Network {
 		m_socket = socket(PF_INET, SOCK_STREAM, 0);
 		struct sockaddr_in name;
 		name.sin_family = AF_INET;
+#ifdef __unix__
```

Comment:
This is unnecessary. I'm pretty sure you can just switch it straight up to __builtin_bswap16() without asking if it's windows or not. Windows vs UNIX plays no role in this, it's psp-sdk's issue.